packages/guides-restructured-text/resources/config/guides-restructured-text.php

```diff
@@ -173,6 +175,10 @@
             ),
         ])
         ->set(ConfvalDirective::class)
+        ->set(ConfigurationBlockDirective::class)
+        ->args([
+            '$languageLabels' => param('phpdoc.rst.code_language_labels'),
```

Comment:
I think I'd use "alias" instead of "label": a single language can be aliased to many different labels. I don't think the opposite needs to be true.

Reply:
We only support one tab per language